src/locales/en.json

```
@@ -82,6 +82,7 @@
     "enable_exposure_notifications_body": "You must enable exposure notifcations to report a positive test result.",
     "enable_exposure_notifications_title": "Enable exposure notifications to continue",
     "error": {
+      "invalid_format": "Verification code can only contain numbers",
```

```suggestion
      "invalid_format": "Invalid format",
```
Mostly just worried about translations being more that one line in length. so fewer words > more words as a rule of thumb.